ai_presenter/config/ai.py

```diff
@@ -1,6 +1,9 @@
 from ai_presenter.config.chatgpt import ChatGPTConfig
+import ai_presenter.config.defaults
 
 
 class AIConfig:
     def __init__(self, data):
+        ai_presenter.config.defaults.TEXT_AI_FILE = data['text_ai_filename']
```

Comment:
Use the style:


```python

from ai_presenter.config.defaults import TEXT_AI_FILE, VOICE_AI_FILE

...

    def __init__(self, data):
        try:
            self.text_ai_filename = data['text_ai_filename']
        except:
            self.text_ai_filename = TEXT_AI_FILE

        try:
            ai_presenter.config.defaults.VOICE_AI_FILE = data['voice_ai_filename']
        except:
            self.voice_ai_filename = VOICE_AI_FILE
```

This model can handle the situation if the values are missing.